kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/schemaregistry/MessageFormatter.java

```diff
@@ -2,4 +2,8 @@
 
 public interface MessageFormatter {
   String format(String topic, byte[] value);
+
+  default MessageFormat getFormat() {
+    return null;
```

Comment:
maybe UNKNOWN would be better here?

Reply:
There is no UNKNOWN enum value yet, I will add it

Reply:
Done

Reply:
I'd use String instead